lib/turbo_boost/commands/runner.rb

```diff
@@ -336,10 +364,15 @@ def append_response_header(key, value)
     controller.response.set_header key.to_s, value.to_s
   end
 
-  def append_to_response_headers(headers = {})
+  def append_to_response_headers(status = nil)
     return unless command_performed?
-    headers.each { |key, val| append_response_header key, val }
-    append_response_header "TurboBoost-Command", command_name
-    append_response_header "TurboBoost-Command-Status", "HTTP #{controller.response.status} #{TurboBoost::Commands::HTTP_STATUS_CODES[controller.response.status]}"
+
+    values = [
+      status || "#{controller.response.status} #{TurboBoost::Commands::HTTP_STATUS_CODES[controller.response.status]}".delete(","),
```

Comment:
Curious question: why `delete(",")`? Looks a bit like a smell at first glance.

Reply:
Just a safeguard as I use `,` as the value delimiter for multiple values stored under the single HTTP header now.

Reply:
Ensures we can properly tokenize those values.